waku/node/peer_manager/peer_manager.nim

```diff
@@ -73,7 +73,7 @@ const
 
 type PeerManager* = ref object of RootObj
   switch*: Switch
-  peerStore*: PeerStore
+  peerStore*: WakuPeerStore
```

Comment:
I wonder if we should also remane the variable name. Maybe do the same in the tests
```suggestion
  wakuPeerStore*: WakuPeerStore
```